views/index.erb

```diff
@@ -0,0 +1,7 @@
+<html>
```

Comment:
'index' and 'index1' are not very descriptive file names, this will make it more difficult for others to figure out what the file is for